powerauth-data-adapter/src/main/resources/static/resources/messages_en.properties

```diff
@@ -1,4 +1,5 @@
 login.smsText=Authorization code for login is {0}.
-login_2fa.smsText=Authorization code for login is {0}.
+login_sca.smsText=Authorization code for login is {0}.
```

Comment:
Keeping same text? `..this login step..`

It there are two separate keys there should be two different messages.

Reply:
The keys are used based on operation name, which is `login` and `login_sca`. This is why multiple keys are used.

Reply:
See sample implementation (most likely also used by clients):
https://github.com/wultra/powerauth-webflow-customization/blob/develop/powerauth-data-adapter/src/main/java/io/getlime/security/powerauth/app/dataadapter/impl/service/DataAdapterService.java#L227